docs/source/guide/pt-1-intro.md

```diff
@@ -50,56 +52,91 @@ the circuit on a noisy simulator, and returns the probability of the ground
 state. See the [Executors](executors.md) section for more information on
 how to define more advanced executors.
 
+During execution by the simulator, a coherent error is introduced by applying a 
+rotation around the X-axis (Rx gate) to each output of any 2-qubit gate in the circuit of interest.
+
+For the sake of this example executed by a simulator, we set the noise level to be proportional to the angle of the Rx rotation.
+
+This noise model is well-suited to highlight the effect of Pauli Twirling,
+which is a technique that transforms coherent noise into incoherent noise.
+
 ```{code-cell} ipython3
-import numpy as np
-from cirq import DensityMatrixSimulator, amplitude_damp
-from mitiq.interface import convert_to_mitiq
+from numpy import pi
+from cirq import CircuitOperation, CXPowGate, CZPowGate, DensityMatrixSimulator, Rx
+from cirq.devices.noise_model import GateSubstitutionNoiseModel
 
-def execute(circuit, noise_level=0.1):
-    """Returns Tr[ρ |0⟩⟨0|] where ρ is the state prepared by the circuit
-    executed with amplitude damping noise.
+def get_noise_model(noise_level: float) -> GateSubstitutionNoiseModel:
+    """Substitute each CZ and CNOT gate in the circuit
+    with the gate itself followed by an Rx rotation on the output qubits.
     """
-    # Replace with code based on your frontend and backend.
-    mitiq_circuit, _ = convert_to_mitiq(circuit)
-    noisy_circuit = mitiq_circuit.with_noise(amplitude_damp(gamma=noise_level))
-    rho = DensityMatrixSimulator().simulate(noisy_circuit).final_density_matrix
-    return rho[0, 0].real
+    rads = pi / 2 * noise_level
+    def noisy_c_gate(op):
+        if isinstance(op.gate, (CZPowGate, CXPowGate)):
+            return CircuitOperation(
+                Circuit(
+                    op.gate.on(*op.qubits), 
+                    Rx(rads=rads).on_each(op.qubits),
+                ).freeze())
+        return op
+
+    return GateSubstitutionNoiseModel(noisy_c_gate)
+
+def execute(circuit: Circuit, noise_level: float):
+    """Returns Tr[ρ |0⟩⟨0|] where ρ is the state prepared by the circuit."""
+    return (
+        DensityMatrixSimulator(noise=get_noise_model(noise_level=noise_level))
+        .simulate(circuit)
+        .final_density_matrix[0, 0]
+        .real
+    )
 ```
 
 The [executor](executors.md) can be used to evaluate noisy (unmitigated)
 expectation values.
 
 ```{code-cell} ipython3
-# Compute the expectation value of the |0><0| observable.
-noisy_value = execute(circuit)
+# Set the intensity of the noise
+NOISE_LEVEL = 0.1
```

Comment:
Good idea setting this explicitly to avoid relying on default values that are easy to miss when reading.